packages/rtk-query-codegen-openapi/ChangeLog.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to the `RTK Query - Code Generator` for `Open API` project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.1.3] - 10-11-2023
+
+### Added
+- Support for Read Only Properties in the Open API spec. Previously, this property was ignored.   
+ Now if the readOnly property is present and set to `true` in a schema, it will split the type into two types: one with the read only property suffixed as 'Read' and the other without the read only properties, using the same type name as before.
+This may cause issues if you had your OpenAPI spec properly typed/configured, as it will remove the read onyl types from your existing type. You will need to switch to the new type suffixed as 'Read' to avoid missing property names.
```

Comment:
There is a bit more of a change :) 

Let me try to get it together:

## 1.1.0 
* Adds the option of generating real TS enums instead of string unions #2854 
* Also ensures compatibility with TypeScript 5.0 versions - as a result also needs a higher TypeScript version to work with (old version range was 4.1-4.5)
* Also changes depenendcy from a temporarily patched old version of `oazapfts` back to the current upstream version

## 1.1.1
* Codegen: better handling of duplicate param names (https://github.com/reduxjs/redux-toolkit/pull/3780)
If a parameter name is both used in a query and a parameter, it will be prefixed with `query`/`param` now to avoid conflicts

## 1.1.2
* Support OpenAPI readOnly/writeOnly properties (https://github.com/reduxjs/redux-toolkit/pull/3430)

## 1.1.3
* adds a temporary workaround for https://github.com/oazapfts/oazapfts/issues/491